### PR TITLE
chore: push aio image to quay

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -68,3 +68,12 @@ jobs:
         tags: ${{ steps.build.outputs.tags }}
         username: ${{ secrets.QUAY_USERNAME }}
         password: ${{ secrets.QUAY_ROBOT_TOKEN }}
+
+    - name: Push aio to quay.io
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        registry: "quay.io/${{ env.quay_org }}"
+        image: aio
```
Comment (nullr0ute) [inline .github/workflows/cd.yml:76]: do we want this to be called something like fdo-aio?

Comment (runcom) [reply .github/workflows/cd.yml:76]: last time with patrick we said fdo was a repetition - perhaps `aio-server`?

Comment (nullr0ute) [reply .github/workflows/cd.yml:76]: if the org is fdo I think just aio is fine

**Review by nullr0ute — APPROVED** [review]
LGTM

**Pull request merged**